### PR TITLE
Echo server demo that receives messages from a socket and echos them back to the client

### PR DESCRIPTION
**Pull request by PramodSSImmaneni**
.The application uses two operators for receiving messages and responding to them

### CODE REVIEW

File: demos/echoserver/pom.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  </build>
+
+  <dependencies>
+    <!-- add your dependencies here -->
+    <dependency>
+      <groupId>com.datatorrent</groupId>
+      <artifactId>malhar-library</artifactId>
+      <version>${datatorrent.version}</version>
+      <!-- 
+           If you know your application do not need the transitive dependencies that are pulled in by malhar-library,
```
Comment (243826) [inline demos/echoserver/pom.xml]: someone please fix the root cause of this typo and all the manifestations.

File: demos/echoserver/src/main/java/com/datatorrent/demos/echoserver/Message.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datatorrent.demos.echoserver;
+
+import java.net.SocketAddress;
+
+/**
+ * Created by Pramod Immaneni <pramod@datatorrent.com> on 12/12/14.
```
Comment (243826) [inline demos/echoserver/src/main/java/com/datatorrent/demos/echoserver/Message.java]: remove please.

File: demos/echoserver/pom.xml (hunk 1 above)
Comment (PramodSSImmaneni) [inline demos/echoserver/pom.xml]: Fixed it locally. Will let concerned person know about fixing it in the template.

File: demos/echoserver/src/main/java/com/datatorrent/demos/echoserver/Message.java (hunk 2 above)
Comment (PramodSSImmaneni) [inline demos/echoserver/src/main/java/com/datatorrent/demos/echoserver/Message.java]: Gone

**Pull request merged**